src/main/resources/app.conf

```diff
@@ -170,18 +186,22 @@ app {
   permissions {
     retry {
       # the retry strategy to use; possible values are: "never", "once" and "exponential"
-      strategy = "exponential"
+      strategy = ${app.sourcing.retry.strategy}
       strategy = ${?PERMISSIONS_RETRY_STRATEGY}
       # the initial delay before retrying that will be multiplied with the 'factor' for each attempt
       # (applicable only for strategy "exponential")
-      initial-delay = 100 milliseconds
+      initial-delay = ${app.sourcing.retry.initial-delay}
       initial-delay = ${?PERMISSIONS_RETRY_INITIAL_DELAY}
+      # the maximum delay applied to the retrying strategy
+      max-delay = ${app.sourcing.retry.max-delay}
```

Comment:
this is not a sourcing value, but the retry mechanism for updating permissions; apply a constant value.

and add env override: `PERMISSIONS_RETRY_MAX_DELAY`